README.md

```diff
@@ -60,16 +66,20 @@ sudo apt install libsystemd-dev gcc pkg-config python3-dev
 
 Then you can install Yin-Yang in a python virtual environment:
 ```bash
+# bash is necessary to run the source command
 bash
+# Removes any already present Yin-Yang code
 rm -rf Yin-Yang
+# Clones the code to your local machine
 git clone https://github.com/oskarsh/Yin-Yang
+# Enters the directory containing Yin-Yang's code
 if pwd != "Yin-Yang"; then cd Yin-Yang; fi
-## Create virtual environment for pypi packages
+## Creates a virtual environment for pypi (pip) packages
 python3 -m venv .venv
 source .venv/bin/activate
```

Comment:
Perhaps, per https://superuser.com/questions/46139/what-does-source-do#comment781187_46146, `source` should become `.`?

Reply:
The documentation for `venv` uses `source`, so I think this is fine.

https://docs.python.org/3/library/venv.html